terraria/helm/terraria/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: terraria
+description: helm chart for terraria
+type: application
+version: 0.1.2
```

Comment:
Do i need to bump this version if I have already published it?

Reply:
The version needs to not have been pushed to Plural yet.